code/style.css

```diff
@@ -1,7 +1,274 @@
+  } 
+
+
+/* Main content */
+
+.main-container {
+  display: flex;
+  margin-top: 800px;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  
+}
+
+
+.form {
+  display: flex;
+  width: 90%;
+  box-sizing: border-box;
+  flex-direction: column;
+  background-color:rgb(118, 216, 230);
+  margin: 20px auto;
+  padding: 20px;
+  border-radius: 10px;
+  box-shadow: 10px 10px 5px #adfff3;
+}
+  
+.form-group {
+  margin-bottom: 15px;
+  display: flex;
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 15px;
+}
+
+.form-group label {
+  width: 100px;
+  margin-right: 5px;
+}
+
+.form-group input,
+.form-group textarea {
+  width: 90%;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  background-color: wheat;
+  border-radius: 3px;
+}
+
+.form-submit {
+  width: 100px;
+  position: relative;
+} 
 
 .active + .answer {
-  /* some styling here */
+  display: block;
+}
+
+.label {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 15px;
+}
+
+input:valid {
+  background-color: rgb(144, 231, 167);
+}
+
+input:invalid {
+  background-color: cornsilk;
+}
+
+button {
+  display: inline-block;
+  padding: 5px 10px;
+  font-size: 12px;
+  cursor: pointer;
+  text-align: center;
+  text-decoration: none;
+  outline: none;
+  color:black;
+  background-color: rgb(42, 197, 218);
+  border: none;
+  border-radius: 10px;
+  box-shadow: 0 4px #999;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  margin: 15px;
+} 
+
+button:hover {
+  background-color: rgb(7, 207, 233);
+}
+
+button:active {
+  background-color: #3e8e41;
+  box-shadow: 0 5px #666;
+  transform: translateY(4px);
+} 
+
+.radio-btn {
+  display: flex;
+  justify-items: self-end;
+  margin: 5px;
+  cursor: pointer;
+}
+
+.checkbox {
+  display: flex;
+  justify-items: self-end;
+  margin: 5px;
+  cursor: pointer;
+}
+
+/* Accordion styling */
+
+.accordion {
+  background-color: #adfff3;
+  box-sizing: border-box;
+  width: 90%;
+  /* width: 40%; */
+  margin: 20px auto;
+  padding: 20px;
+  border-radius: 10px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 15px bold;
+  border-radius: 10px;
+  box-shadow: 10px 10px 5px rgb(58, 190, 207) ;
+}
+
+.question {
+display: flex;
+margin-bottom: 10px;
+border: 3px solid rgb(7, 207, 233);
+border-radius: 3px;
+padding: 7px;
+}
+
+.answer {
+display: none;
+margin-bottom: 5px;
+background-color: rgb(129, 226, 230);
+border: 3px solid rgb(126, 208, 219);
+border-radius: 3px;
+padding: 3px;
+}
+
+
+/* footer styling */
+
+.footer {
+    position: relative;
+    height: 50px;
+    padding-top: 20px;
+    margin-top: 20px;
+    width: 80%;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    background-color:#adfff3; 
+    color: rgb(54, 47, 47);
+    text-align: center;
+  }
+
+
+/* syns i mobil vy */
+
+  @media (max-width: 767px){
+  
+    .top-section-container{
+      background-size: 100% 100%;
+      width: 100%;
+      height: 100%;
+    }
```

Comment:
This need a peak since it's not responsive - the image pushes together and this need a solution before you're approved!

Reply:
Yes, changed the background-size to cover and changed the font-size to a smaller one in mobile view to make it more responsive, hope that dose it =)

Reply:
I made all my code changes in my code (in the VS-editor) and pushed them again to Git, is that enough or should I do something else/more? plz let me know if so.